specs/2019.json

```diff
@@ -17471,7 +17471,7 @@
       "rightSensoryZpp":"T1",
       "leftSensoryZpp":"S1",
       "rightMotorZpp":"C8*",
-      "leftMotorZpp":"L4*,L5*"
+      "leftMotorZpp":"L4,L5*"

```

Comment:
This case broke after the logic was updated. `L4` should not have been flagged with `*` as no `*` value affects it. I've reviewed this case with Kris Walden.